docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Listview.htm

```diff
@@ -53,9 +53,11 @@ <h2><a id="manual-constructor"></a>Manual Constructor</h2>
    &lt;li&gt;Barry&lt;/li&gt;
    &lt;li&gt;Bill&lt;/li&gt;
 &lt;/ul&gt;
-&lt;script&gt;
-   tau.widget.Listview(document.getElementById(&quot;list&quot;));
-&lt;/script&gt;
+</pre>
+<p>JS code:</p>
```

Comment:
Global Suggestion : How about adding "\<p>HTML code:\</p>" as "\<p>JS code:\</p>" added?

If you want to keep current state, I'll merge this PR.